academy/4-my-own-chain/game-wager.md

```diff
@@ -401,7 +381,9 @@ With the desired steps defined in the wager handling functions, it is time to in
     }
     ```
 
-## ~~Unit~~ Integration tests
+<!-- Changed title from: "## ~~Unit~~ Integration tests" -->
+
+## Integration tests
```

Comment:
The idea was to show that at this stage unit tests were no longer working. It showed ugly on the side. @coldice

Reply:
If we do want to do that, we still need at least one sentence after a heading. Maybe a highlightbox would be better for this case. This way it looks like something went missing.

Reply:
@xavierlepretre please address before merge.

Reply:
Are you saying that strikethrough makes it look like something was missing?

Reply:
Not 100%. The strikethrough just doesn't display correctly in the navigation. But having the title and nothing else did not signal for me that unit tests were no longer working but rather gave me the impression someone forgot to include the content. I think we should change how we want to pass along the information.

Reply:
👍